_articles/bluetooth.md

```diff
@@ -107,7 +167,7 @@ Then they can be installed with this command:
 sudo dpkg -i linux-firmware_1.167.1_all.deb
```

Comment:
double-check this is up to date, I think it is but sometimes these things change.

Reply:
addressed.